op-monitorism/faultproof_withdrawals/monitor.go

```diff
@@ -98,17 +95,107 @@ func NewMonitor(ctx context.Context, log log.Logger, m metrics.Factory, cfg CLIC
 
 		maxBlockRange: cfg.EventBlockRange,
 
-		state:   *state,
+		state:   State{},
 		metrics: *metrics,
 	}
 
+	// is starting block is set it takes precedence
+	startingL1BlockHeight := cfg.StartingL1BlockHeight
+	hoursInThePastToStartFrom := cfg.HoursInThePastToStartFrom
+
+	// In this case StartingL1BlockHeight is not set
+	if startingL1BlockHeight == 0 {
+		// in this case is not set how many hours in the past to start from, we use default value that is 14 days.
+		if hoursInThePastToStartFrom == 0 {
+			hoursInThePastToStartFrom = DefaultHoursInThePastToStartFrom
```

Comment:
Off by one here seems if we really want to start at zero we should maybe mention into the doc?

Reply:
Solved